server/routes/approvedVisitors.js

```diff
@@ -0,0 +1,56 @@
+const express = require('express');
+
+const createApprovedVisitorsRouter = ({ offenderService }) => {
+  const router = express.Router();
+
+  const getPersonalisation = async (user, query) => {
+    const rawApprovedVisitors = await offenderService.getApprovedVisitorsFor(
+      user,
+    );
+    const { page: rawPage = 1 } = query;
+    const maxVisitorsPerPage = 10;
+    const totalCount = rawApprovedVisitors.length;
+    const page =
+      Number.isNaN(rawPage) || rawPage > totalCount ? 1 : Number(rawPage);
```

Comment:
should this be `rawPage > totalPages` rather than `totalCount`?

meh.. NBD but I'd have shoved this in a util as the concept of "providing pagination over a list" is pretty generic (non domain specific) concept that could be unit tested in isolation outside of having all the gumpf of testing within a rest controller